File: app/src/main/java/com/example/test1/MainViewModel.java
package com.example.test1;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.LiveData;

import com.example.test1.model.Data;
import com.example.test1.model.PostRepository;
import java.util.List;

public class MainViewModel extends AndroidViewModel {
  private PostRepository postRepository;

  public MainViewModel(@NonNull Application application) {
    super(application);
    postRepository = new PostRepository();
  }

  public LiveData<List<Data>> getAllEmployee() {
    return postRepository.getMutableLiveData();
  }
}

File: app/src/main/java/com/example/test1/MainViewModel.kt
package com.example.test1

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.LiveData
import com.example.test1.model.Data
import com.example.test1.model.PostRepository

class MainViewModel(application: Application) : AndroidViewModel(application) {
    private val postRepository: PostRepository
    val allEmployee: LiveData<List<Data>>
        get() = postRepository.getMutableLiveData()

    init {
        postRepository = PostRepository()
    }
}